mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (order) {
```

Comment:
This conditional check is not necessary. The `findByIdAndRemove` method returns `null` if no document is found, so the `if (order)` check is redundant. It can be removed to simplify the code.